app/views/items/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_tag("/merchants/#{params[:merchant_id]}/items", method: :post) do %>
+  <p>Item name:</p>
+  <%= text_field_tag :name %>
+  <p>Item description:</p>
+  <%= text_field_tag :description %>
+  <p>Item price:</p>
+  <%= text_field_tag :price %>
+  <p>Item image link:</p>
+  <%= text_field_tag :image %>
+  <p>Item status:</p>
+  <%= text_field_tag :status %>
```

Comment:
This is what I was referring to in one of my previous comments.. If you had made status a boolean that was defaulted to true when creating an item, this would not need to be a field for a user to fill out... because who knows what they'll put in there 🤷‍♀ .